docs/getting-started-guides/ubuntu/operational-considerations.md

```diff
@@ -115,8 +115,8 @@ juju switch default
 
 ### Running privileged containers
 
-By default, juju-deployed clusters do not support running privileged containers.
-If you need them, you have to enable the ```allow-privileged``` config on both
+By default, juju-deployed clusters only allow running privileged containers on nodes with GPUs.
+If you need privileged containers on other workers, you have to enable the ```allow-privileged``` config on both
```

Comment:
can we use the same term for consistency here? Either `workers` or `nodes` (my preference is `nodes`, but YMMV), Just don't want to use two terms for the same thing in adjoining sentences...